lib/util.rb

```diff
@@ -20,6 +20,14 @@ def self.rootish_ssh(host, user, keys, cmd)
     end
   end
 
+  def self.create_root_certificate(common_name:, duration:)
+    create_certificate(
+      subject: "/C=US/O=Ubicloud/CN=#{common_name} Root Certificate Authority",
```

Comment:
You shouldn't add " Root Certificate Authority" anymore since you are getting common_name. Caller would add that if they really want it, which they may not want it in some cases.